src/node/interfaces.cpp

```diff
@@ -212,9 +212,10 @@ class NodeImpl : public Node
     bool getHeaderTip(int& height, int64_t& block_time) override
     {
         LOCK(::cs_main);
-        if (::pindexBestHeader) {
-            height = ::pindexBestHeader->nHeight;
-            block_time = ::pindexBestHeader->GetBlockTime();
+        auto best_header = chainman().pindexBestHeader;
```

Comment:
first commit: You are touching all lines where `pindexBestHeader` is used. It seems a scripted-diff can be appended to rename it to `m_best_header` without causing additional merge conflicts.

Reply:
Done ddd6bafc16e75e1fb93fd0177f5c5c7c64579095